kubernetes/annotations_and_labels.md

```diff
@@ -20,6 +23,9 @@ This page defines common annotations and labels we set in Kubernetes objects.
   Secrets and CertConfigs.
 - `giantswarm.io/cluster` - value should contain tenant cluster ID which this
   object is part of. E.g. `giantswarm.io/cluster=eggs2`.
+- `giantswarm.io/machine-deployment` - value should contain tenant cluster node
```

Comment:
This is managed by `api` atm..

Reply:
All this should sounds weird but we have it in other descriptions already.